### DISCUSSION TITLE
大佬  清晰度咋没4K和1080P60帧的选项了！！！

### DISCUSSION THREAD

**Discussion by a48150908** [Richasy/Bili.Uwp#831] [常规]
大佬  清晰度咋没4K和1080P60帧的选项了！！！

**Comment by XiaoMouz** [comment]
我的还在，是不是你大会员过期了:(
![image](https://user-images.githubusercontent.com/54032212/158286418-9596caec-afbf-4786-8e8a-d7fc3f740529.png)
